geps/gep-1619.md

```diff
@@ -159,47 +167,487 @@ Generally, the implementation API programs the dataplane API; however these two
 
 Java application servers such as Tomcat and Jetty, were the first to standardize the API around persistent sessions. These Java applications introduced the “jsessionid” cookie as well as more advanced features such as session migration, replication, and on demand session activation. It’s important for Gateway API to examine persistent session use cases and history from Java APIs to ensure the API is designed appropriately.
 
+### Use Cases
```

Comment:
I'm trying to decide how to reconcile Use Cases here with my new section on [`Expected API Behavior`](https://github.com/kubernetes-sigs/gateway-api/pull/2159/files#diff-afa0e827962fe48152635217c6bc954020d5fec3578af4a9e5f744a2b4568fffR533). Use Cases here aren't supposed to be prescriptive, but more or less informational. However, since the suggestions that this spec shouldn't describe implementation details for implementations, I don't know if these Use Cases are even necessary.

They feel helpful, but also are probably too low-level considering the suggestions that implementations should decide on how to implement session persistence.

TBD